doc/toolbox-enter.1.md

```diff
@@ -4,23 +4,24 @@
 toolbox\-enter - Enter a toolbox container for interactive use
 
 ## SYNOPSIS
-**toolbox enter** [*--container NAME* | *-c NAME*]
-              [*--release RELEASE* | *-r RELEASE*]
+**toolbox enter** [*OPTIONS*] \<*CONTAINER*\>
 
 ## DESCRIPTION
 
-Spawns an interactive shell inside a toolbox container. The container should
-have been created using the `toolbox create` command. If there aren't any
-containers, `toolbox enter` will offer to create one for you. When invoked with
-the default parameters, and if there's only one container available, it will
-fall back to it, even if it doesn't match the default name.
+Spawns an interactive shell inside a toolbox container. `toolbox enter` is built
+on top of **toolbox run** by executing a shell (default is the shell used on the
```

Comment:
The manuals are meant to be user-facing, so I don't know if we should go so far as to say that `enter` is implemented on top of `run`. It seems like implementation details.

However, it's definitely useful to mention which shell is chosen and the fallback to `/bin/bash`.

Reply:
I split this out into https://github.com/containers/toolbox/pull/682